extensions-core/kafka-extraction-namespace/src/main/java/org/apache/druid/query/lookup/KafkaLookupExtractorFactory.java

```diff
@@ -262,7 +262,7 @@ public void onFailure(Throwable t)
               }
             }
           },
-          Execs.directExecutor()
+          Runnable::run
```

Comment:
I think it's better to create a method like `GuavaUtils.directExecutor()` which returns `Runnable::run`. Bare `Runnable::run` looks somewhat puzzling. There are also other calls to `Runnable::run` in this PR which are subject for this.